### PR TITLE
Replacing a few more Keybase links

### PR DESCRIPTION
**Pull request by stopfeds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ See further details on the [this wiki page](https://bisq.wiki/Growth_Team#Proces
 
 All translations work, for the website and otherwise, takes place on [Transifex](https://www.transifex.com/bisq/). Please do not make pull requests for non-English text changes in this repository!
 
-If you'd like to propose adding a new language to the website, please get in touch in the #transifex channel [on Keybase](https://keybase.io/team/bisq) or open an issue in this repository.
+If you'd like to propose adding a new language to the website, please get in touch in the Growth channel [on Matrix](https://bisq.chat/) or open an issue in this repository.
```
Comment (m52go) [inline README.md]:
```suggestion
If you'd like to propose adding a new language to the website, please get in touch in the Growth channel [on Matrix](https://matrix.to/#/#bisq.growth:bitcoin.kyoto) or open an issue in this repository.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -12,8 +12,7 @@ please make sure you've corresponded with the repo's maintainer to
 ensure it's work that is currently prioritized and budgeted.
 
 You can do this by opening an issue explaining your proposed changes and
-cost, or by floating the idea in the #website channel on 
-Keybase (https://keybase.io/team/bisq).
+cost, or by floating the idea in the Growth channel on Matrix (https://bisq.chat).
```
Comment (m52go) [inline .github/pull_request_template.md]:
```suggestion
cost, or by floating the idea in the Growth channel on Matrix (https://matrix.to/#/#bisq.growth:bitcoin.kyoto).
```

**Review by m52go — COMMENTED** [review]
Thanks -- I think it might be better to link the growth channel directly where it's mentioned. What do you think?

File: .github/pull_request_template.md (hunk 2 above)
Comment (stopfeds) [reply .github/pull_request_template.md]: Yes, makes sense. Didn't know that this works.

**Pull request merged**